uint32                              id
string                              reason
string                              job_id